SPCTR-3180: Fix: CTA - Resolved alignment issue for buttons in call to action. (#3463)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Fix: Call To Action - Resolved an issue wherein the alignment of buttons is not working as intended.

Reviewer: ```suggestion
* Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
```

Author: Done.
